metrics.go

```diff
@@ -247,6 +247,7 @@ var (
 			"model_family",
 			"model_name",
 			"serial_number",
+			"message",
```

Comment:
Labeling the metric by an essentially arbitrary string is not the Prometheus way. See "Do not overuse labels" in the [documentation](https://prometheus.io/docs/practices/instrumentation/#things-to-watch-out-for). You're basically trying to put logs into Prometheus – but Prometheus is not a logging system.

Reply:
Sounds reasonable, thanks!
I'll remove it.

Reply:
@lahwaacz done